lib/src/parser.ts

```diff
@@ -238,7 +238,8 @@ function extractEndpoint(
               ? undefined
               : nextNonNamePositionRelative
           ),
-          type: VOID
+          type: VOID,
+          description: 'TODO'
```

Comment:
Instead of `TODO`, could we make this an empty string?

Reply:
Yeap